utilities/utilities.css

```diff
@@ -1,168 +1,170 @@
 /* Textbox */
 
 .textbox {
-
-    /* border: solid 1px black; */
-    /* padding-left: 24px;
+  /* border: solid 1px black; */
+  /* padding-left: 24px;
     padding-right: 24px; */
+}
+
+.title {
+  font-size: 21px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 32px;
+  letter-spacing: 0px;
+  text-align: left;
+  margin-bottom: 0px;
+  margin-top: 10px;
+}
+.subtitle {
+  margin-bottom: 10px;
+  margin-top: 0px;
+  color: var(--font-light);
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 20px;
```

Comment:
should not use units for line-height, 
For example, you can directly give multipliers like
line-height: 1.5

Reference: https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#accessibility_concerns

Reply:
Yes thanks btw